# /control/parameters topic 
bool change
int32 value
